packages/terra-data-grid/CHANGELOG.md

```diff
@@ -10,7 +10,10 @@
 ## 1.13.0 - (February 16, 2024)
 
 * Changed
- * Removed the default rowMinimumHeight from FlowsheetDataGrid so that the `terra-table` default value is used.  
+ * Removed the default rowMinimumHeight from FlowsheetDataGrid so that the `terra-table` default value is used.
+
+* Added
+  * Column actions support added.
```

Comment:
nit (for consistency):
```suggestion
  * Added support for column actions.
```

Reply:
Updated in https://github.com/cerner/terra-framework/pull/2024/commits/7dec53891e0f3c52978a589d7f79fcbbd448103d